clustered/client/src/test/java/org/ehcache/clustered/client/replication/ActivePassiveClientIdTest.java

```diff
@@ -190,31 +186,29 @@ public void trackedMessagesReplicatedToPassive() throws Exception {
     passiveEntity = observableClusterTierServerEntityService.getServedPassiveEntities().get(1);
     passiveMessageHandler = passiveEntity.getMessageHandler();
 
-    Set<ClientSourceId> clientSourceIds = passiveMessageHandler.getTrackedClients();
-    assertThat(clientSourceIds).hasSize(1); // one client tracked
+    Assert.assertThat(passiveMessageHandler.getTrackedClients().count(), is(1L)); // one client tracked
```

Comment:
and since this assertion is pretty popular, I guess you could refactor it in its own method. Doing `assertThat(passiveMessageHandler.getTrackerClients().count()).is(x)`